### DISCUSSION TITLE
Bad text display

### DISCUSSION THREAD

**Discussion by DKGH** [ElinamLLC/SharpVectors#282] [Q&A]
I got the same SVG file which appears different in SharpVectors. This is how it looks like in Chrome:
![view_from_chrome](https://github.com/ElinamLLC/SharpVectors/assets/16451561/736c6df9-b25d-49fc-b238-b66e6d39cdb9)

And this is how it looks in the sample project: WpfTestSvgControl:
![view_from_sharp_vectors](https://github.com/ElinamLLC/SharpVectors/assets/16451561/3e9cab5e-a6a4-4556-8705-7ea8bae330c8)

What could be the reason? I have attached the original svg.
![demo01](https://github.com/ElinamLLC/SharpVectors/assets/16451561/5673e94a-ed70-40bf-b3df-34daa1d6bc29)

**Comment by paulushub** [comment]
The text support improvement is a pending task - still pending mainly because WPF died!

**Reply by DKGH** [reply]
Is there another sample besides WpfTestSvgControl that I should be using?

**Reply by paulushub** [reply]
`WpfTestSvgControl` is a sample and in this case for hit test. I used `WpfSvgTestBox` for debugging, and there are other samples.
However, none of these will change the output since the issue is with the rendering engine, not the viewer.

**Reply by DKGH** [reply]
Yes, I noticed other skia based control are also doing the same thing. They seem to be ignoring the dominant-baseline="hanging" attribute. Also they are all using bold font-weight. This seems unalterable even when normal font-weight is specified.

**Reply by paulushub** [reply]
> dominant-baseline="hanging" 

That is a new SVG option introduced in SVG-2, and in the MDN docs below, it is not known yet which browsers support it.
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/dominant-baseline

So, the only way is to support it on the rendering engines, be it SharpVectors, Skia.Svg or the others.
If you need only the static image output, you can also try resvg (a rust implementation) or the browser:
https://github.com/RazrFalcon/resvg

**Reply by paulushub** [reply]
@Suplanus Sorry for the delay.
By the latest commit, it seems I have resolved this issue.
Please can you verify this?
